File: src/main/java/org/jdc/template/log/JavaTree.java
package org.jdc.template.log;

import timber.log.Timber;

public class JavaTree extends Timber.Tree {
    @Override
    protected void log(int priority, String tag, String message, Throwable t) {
        String logMessage;
        if (tag != null && !tag.isEmpty()) {
            logMessage = "[" + tag + "] " + message;
        } else {
            logMessage = message;
        }

        System.out.println(logMessage);
    }
}

File: app/src/main/kotlin/org/jdc/template/log/JavaTree.kt
package org.jdc.template.log

import timber.log.Timber

class JavaTree : Timber.Tree() {
    override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
        val logMessage = if (tag != null && tag.isNotEmpty()) {
            "[$tag] $message"
        } else {
            message
        }

        println(logMessage)
    }
}
